New design

iforgotbetter opened the pull request:
So I got a little creative...

I removed the old icons in the footer and added a whole new footer. Icons are free, gave credit in footer. Added some images to the assets. Added some meta information to the head. Cleaned up the html structure. Added some imports to the style.scss from Bulma.

The yarn run dev script needs to be fixed. I noticed a problem when I pulled for the first time. On @HiDeoo's commit #820cea7 he added the mkdir. However this only created a new issue when you want to run the script again. It will throw error because the directory already exists. I do not know how to solve this with cli. Somehow delete the old dist directory, or check if exists and just copy contents. Not a huge deal really tho.

Hope you like it :) Thanks for letting me contribute. Excited for beta release.

Commit:
Changed href links in header to properly link to files.

Commit:
Added discord icon font. Added yard script to move fonts dir. Added include in the custom css becuase of variable errors. Need to add a fonts folder in the dist folder, or create that folder in the yarn script.

Commit:
Removed that font script in yarn that was added in the last commit.

Commit:
Removed custom font icons.

Commit:
Added the make dir for dist.

Commit:
Added images.

Commit:
Updates.

Commit:
Updates.

iforgotbetter commented:
Conflicts would be resolved by using mine. There is nothing from HiDeoo that I didn't have, or needed.

HiDeoo commented:
Hey, just gonna give my opinion on this PR if this can help a bit, mostly regarding code as I'm not the best to give opinion on admArt ^^ I'm not a huge fan of the shiny icons but besides that, that looks pretty solid but @Adam13531 or Jason would be a better judge than me :D 

First, regarding the PR itself, you should update your fork & branch and fetch the last upstream changes so @Adam13531  can merge it without any conflicts even if you already have all features from master. It's easy & quick on your end and avoid @Adam13531 dealing with conflicts if he wants to merge it, it should be a 1 button click for him.

Regarding the `mkdir` issue, indeed, I always forget that my local `mkdir` is always aliases to `mkdir -p`, that's the proper fix that you should include.

Most important thing, the PR is not usable as is, we can't `yarn install` it, you manually added comments to the `package.json` file which is now invalid.

I've commented directly various code changes in the PR but to add a brief overview to this comment:

- Invalid `package.json`
- Duplicated sass import
- Unnecessary sass import
- Weird inconsistent sass styling specially regarding line breaks & spaces. It should be consistent, at the moment, some new directives or closures are sometime separated by 1 line break, sometime 2 line breaks or even 3 line breaks. I'm not sure if this is supposed to mean something, but if it is, then it should also be applied to the whole file and be consistent. (I didn't comment them all as it's pretty weird to me)

Great job anw 🎉 

PS: One small design change I'm not a huge fan of is that the Bot Land intro video is lower in the page, now even on a pretty large screen, I have to scroll to see it entirely.

HiDeoo commented on assets/sass/custom.scss:
```diff
@@ -1,23 +1,40 @@
-$circuitboardColor: #b6dac1;
-$circuitboardImage: "data:image/svg+xml,%3Csvg width='304' height='304' viewBox='0 0 304 304' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M44.1 224c.463-2.282 2.48-4 4.9-4 2.76 0 5 2.24 5 5s-2.24 5-5 5c-2.42 0-4.437-1.718-4.9-4H0v-2h44.1zm160 48c.463-2.282 2.48-4 4.9-4 2.76 0 5 2.24 5 5s-2.24 5-5 5c-2.42 0-4.437-1.718-4.9-4H82v-2h122.1zm57.8-46c-.463 2.282-2.48 4-4.9 4-2.76 0-5-2.24-5-5s2.24- ...
+// Had an error not loading variables 
+@import "../../node_modules/bulma/sass/utilities/_all";
```
This is already imported and removing it does not cause any error for me.

iforgotbetter replied:
I had an error for `@imports` with variables from Bulma. I agree this is redundant but thankfully sass wont double import.

HiDeoo replied:
Strange, can you repro it? I can't as it's already included in `assets/sass/syle.scss` which includes this file.

HiDeoo commented on assets/sass/custom.scss:
```diff
@@ -66,13 +85,187 @@ $circuitboardImage: "data:image/svg+xml,%3Csvg width='304' height='304' viewBox=
   }
+
+
+
```
3 line breaks ??? 😕

iforgotbetter replied:
Woops.

HiDeoo commented on assets/sass/custom.scss:
```diff
@@ -57,6 +74,8 @@ $circuitboardImage: "data:image/svg+xml,%3Csvg width='304' height='304' viewBox=
 }
 
+
+
```
??? 🤔

HiDeoo commented on assets/sass/style.scss:
```diff
@@ -3,9 +3,11 @@
 @import "../../node_modules/bulma/sass/utilities/initial-variables";
 @import "../../node_modules/bulma/sass/utilities/_all";
 @import "../../node_modules/bulma/sass/base/_all";
+@import "../../node_modules/bulma/sass/grid/_all";
```
This doesn't seems to be used & removing it doesn't seems to change anything to the page.

iforgotbetter replied:
Yes I was trying to use `level` from Bulma. I think I removed all levels.

HiDeoo replied:
I still can see `level` reference in the HTML tho.

HiDeoo commented on package.json:
```diff
@@ -10,7 +10,7 @@
     "devcommands": "yarn run moveimages && yarn run build:css",
     "build": "yarn run devcommands && postcss --use autoprefixer --output dist/style.css dist/style.css",
-    "moveimages": "cp -r assets/images/* dist/",
+    "moveimages": "mkdir dist && cp -r assets/images/* dist/", // this is not setup properly to handle yarn run dev restarts as it will fail since the dist folder already exists.
```
This is invalid syntax & the proper fix for `mkdir` is `mkdir -p`.

iforgotbetter replied:
Yes sorry I forgot to remove comment as I wanted to leave a note on the PR.

iforgotbetter commented:
@HiDeoo I agree with the conflicts, and making it easier for @Adam13531. However the only change that you made on your last PR and that I didn't was the package.json mkdir. Everything else was html or style which was changed completely.

~The intro video margin wasn't changed. Someone else added 100px margin on class `.description`.~ Solved.

So mkdir -p will fix the issue for restarting `yarn run dev`? Just for my reference.

iforgotbetter commented:
Oh and thank you for taking the time to help fix my errors! @HiDeoo

HiDeoo commented:
No problem, like I said I'm bad at UI or UI review but if I can help with code ^^

Regarding `mkdir`, I'm just gonna quote the `man mkdir` which explains it better than me: `-p: no error if existing, make parent directories as needed` so yeah no error would make it work no matter what.

Regarding the conflicts, like you said it is small conflicts so indeed fetching the last upstream changes on your end will take 30s and add that beautiful green "no merge conflicts" button 😄

iforgotbetter committed:
Update from @hideoo's suggestions.

Fixed spaces & line breaks. Removed import.

iforgotbetter committed:
Updates.

Noticed the 8rem padding bottom that @hideoo mentioned on the intro video. Thank you.

iforgotbetter committed:
Merge branch 'master' into new_design

iforgotbetter commented:
@HiDeoo I agree. Im trying to figure out how to resolve conflicts. I thought I did it, but this PR is still saying there is conflicts.

iforgotbetter committed:
Updates

hiDeoo says this is not needed.

HiDeoo commented:
Looks like you fixes them all :)

Great job on fixing these suggestions I made and removing the merge conflicts 👍

iforgotbetter committed:
Added -p flag to mkdir on `moveimages` script

iforgotbetter committed:
Merge branch 'master' into new_design

iforgotbetter commented:
Saw you added a serve script & analytics tags. Updated my PR to have no merge conflicts.

HiDeoo commented:
Looks like you're still having conflicts, not sure how you're trying to fix them but if you're having issues, tutorials on how to update a fork from upstream for a PR can be easily found on Google.

iforgotbetter committed:
Merge branch 'master' into new_design

HiDeoo commented:
Ho, you made it, nice, great job 👍
